src/components/data-management/SamplesTable.jsx

```diff
@@ -105,26 +103,6 @@ const SamplesTable = forwardRef((props, ref) => {
       dataIndex: 'matrix',
       render: (tableCellData) => <UploadCell columnId='matrix' tableCellData={tableCellData} />,
     },
-    {
```

Comment:
Don't we want to remove the `SpeciesSelector.jsx` and `SpeciesCell.jsx` component files too?  `SpeciesSelector` is only referenced by `SpeciesCell`, which also has been removed as part of this delete in this file

Reply:
No, I didn't delete it because the ticket asks not to delete the actual code.